src/main/java/com/sparta/popupstore/domain/common/util/ValidUtil.java

```diff
@@ -0,0 +1,11 @@
+package com.sparta.popupstore.domain.common.util;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class ValidUtil {
+
+    public static boolean isValidNullAndEmpty(Object obj) {
+        return obj != null && !obj.toString().isEmpty();
+    }
+}
```

Comment:
```suggestion
@UtilityClass
@Component
public class ValidUtil {

    public boolean isValidNullAndEmpty(Object obj) {
        return obj != null && !obj.toString().isEmpty();
    }
}
```
이 클래스도 bean 등록해서 써먹는 게 어떨까요?